djangoapp/requirements.txt

```diff
@@ -0,0 +1,3 @@
+Django
```

Comment:
We should be specific about version numbers in `requirements.txt`, otherwise installs are non-deterministic and people will get different versions at different times:

```
Django==2.0.5
boto3==1.7.13
```